doc/src/getting-started.md

```diff
@@ -0,0 +1,152 @@
+# Getting Started
+
+The hevm project is an implementation of the Ethereum Virtual Machine (EVM)
+focused on symbolic analysis of evm bytecode. This essentially means that hevm
+can try out _all_ execution possibilities of your contract and see it can
+somehow violate some assertions you have, such as e.g. the total number of
```

Comment:
Maybe also mention other errors as well (overflows etc.)

Reply:
Added overflow, good point! I don't know if I wanna put too many here, but maybe we can have a section in `when-to-use.md` ?